translationtable/clinvarxml_alpha.yaml

```diff
@@ -91,7 +92,10 @@
 "Inversion": "inversion"
 "Microsatellite": "microsatellite"
 "protein only": "coding sequence alteration"
-"CompoundHeterozygote": "variant single locus complement"
-# ############################################################
+"CompoundHeterozygote": "compound heterozygous"
+"Haplotype": "haplotype"
+"Phase unknown": "haplotype"
+"Distinct chromosomes": "mosaic"
```

Comment:
is this different than the moasic change in globaltt?

Reply:
This is the same, I changed mosaic to map to :mosaic_genotype to differentiate from the NCIT mosaic https://www.ebi.ac.uk/ols/ontologies/ncit/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FNCIT_C88144

Ideally this would be an SO or GENO term and be categorized as a genotype

Reply:
The more I think about it, mosaicism is not really a genotype but an organism with two distinct genotypes, or a mode of inheritance.  Although I'm unsure how to type this record: https://www.ncbi.nlm.nih.gov/clinvar/variation/101408/